tests/testProcessCcd.py

```diff
@@ -110,26 +110,26 @@ def testProcessCcd(self):
                     print("\nMeasured results:")
                     print("background mean = %r, stdDev = %r" % (bgMean, bgStdDev))
                     print("len(icSrc) :", len(icSrc))
-                    print("len(sources) =", len(sources))
+                    print("len(src) :", len(src))
 
                     print("numGoodPix =", numGoodPix)
                     print("image mean = %r, stdDev = %r" % (imMean, imStdDev))
                     print("variance mean = %r, stdDev = %r" % (varMean, varStdDev))
                     print("psf Ixx = %r, Iyy = %r, Ixy = %r" % (psfIxx, psfIyy, psfIxy))
 
-                    self.assertAlmostEqual(bgMean, 327.61695151065607, places=7)
-                    self.assertAlmostEqual(bgStdDev, 0.40081096398879795, places=7)
+                    self.assertAlmostEqual(bgMean, 327.73706832701379, places=7)
+                    self.assertAlmostEqual(bgStdDev, 0.478942723145386, places=7)
                     self.assertEqual(len(icSrc), 28)
-                    self.assertEqual(len(sources), 188)
-                    self.assertEqual(numGoodPix, 1953514)
+                    self.assertEqual(len(src), 183)
+                    self.assertEqual(numGoodPix, 1966976)
 
-                    self.assertAlmostEqual(imMean,      1.7045581411166495, places=7)
-                    self.assertAlmostEqual(imStdDev,  163.66290334994014, places=7)
+                    self.assertAlmostEqual(imMean,      1.6992922142160001, places=7)
+                    self.assertAlmostEqual(imStdDev,  163.66265133203555, places=7)
                     self.assertAlmostEqual(varMean,   212.5416436210289, places=7)
                     self.assertAlmostEqual(varStdDev, 121.25084470503148, places=7)
-                    self.assertAlmostEqual(psfIxx, 2.790601924907123, places=7)
-                    self.assertAlmostEqual(psfIyy, 2.1989110429762513, places=7)
-                    self.assertAlmostEqual(psfIxy,  0.16599586021207952, places=7)
+                    self.assertAlmostEqual(psfIxx, 2.205197497931509, places=7)
+                    self.assertAlmostEqual(psfIyy, 2.0893651175583723, places=7)
+                    self.assertAlmostEqual(psfIxy,  -0.07647011192069166, places=7)
```

Comment:
That's a HUGE change (half a pixel in width, and different orientation)!  You should verify that it's correct.